kstars/fitsviewer/fitsdata.cpp

```diff
@@ -796,6 +797,96 @@ void FITSData::runningAverageStdDev()
     }
 }
 
+QVector<double> FITSData::createGaussianKernel(int size, double sigma)
```

Comment:
OK, additional filters, nice. I think at some point we need to move filters out of fitsdata too. OK for now.